### PR TITLE
Netwitness App: Bugfix: Fixed a timeframe params formatting issue when provided with the query param in get pcap action

### PR DESCRIPTION
**Pull request by amangal-crest**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
- [x] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: release_notes/unreleased.md [hunk 1]
```diff
@@ -1 +1,3 @@
 **Unreleased**
+
+* Fixed an issue in the get pcap action [PAPP-21456]
```
Comment (paulnawat) [inline release_notes/unreleased.md]: Can we make the release note more specific? (e.g. Fixed wrong formatted params with query and timeframe?)

File: netwitness_connector.py [hunk 2]
```diff
@@ -266,7 +273,7 @@ def _get_capture(self, param, cap_type):
                 except Exception as e:
                     return action_result.set_status(phantom.APP_ERROR, consts.NETWITNESS_INVALID_PARAM.format(message=e))
 
-                query += ' && time="{0}"="{1}"'.format(time1, time2)
+                query += ' && time="{0}"-"{1}"'.format(time1, time2)
```
Comment (paulnawat) [inline netwitness_connector.py:276]: I believe this is the fix, right? 
Should we comment on why it's important to use the symbol `-` over `=`?

**Comment by paulnawat** [comment]
@amangal-crest Changes look great. Could you please update the description of this PR as well? This will let other people who does not have to our Jira to know the description of the issue.

**Review by paulnawat — CHANGES_REQUESTED** [review]
Changes look nice. I left some minor comments.

File: netwitness_connector.py (hunk 2 above)
Comment (amangal-crest) [reply netwitness_connector.py:276]: Yes, this is the fix here. Actually, the '=' was incorrectly placed as the API expects the formatted query to be sent with the '-' sign in between, so it's just a syntactical correction and description for this is already present in the action verbose as mentioned below

NOTE: If start_time and end_time are included along with a query, then the time-frame will be appended to the end of the query. For example: if the query is ip.src=10.10.10.10, the start time is 2018-01-01 00:00:00, and the end time is 2018-01-01 23:59:59, then the final query would be ip.src=10.10.0.7 && time="2018-01-01 00:00:00"-"2018-01-01 23:59:59"

File: release_notes/unreleased.md (hunk 1 above)
Comment (amangal-crest) [reply release_notes/unreleased.md]: Updated

Comment (paulnawat) [reply release_notes/unreleased.md]: super, thanks!

File: netwitness_connector.py (hunk 2 above)
Comment (paulnawat) [reply netwitness_connector.py:276]: I see. thanks!

**Review by paulnawat — DISMISSED** [review]

**Review by cwadhwani-crest — APPROVED** [review]

**Review by ishans-crest — APPROVED** [review]

**Pull request merged**